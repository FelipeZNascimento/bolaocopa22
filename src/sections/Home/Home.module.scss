.app {
  text-align: center;
}

.header {
  background-color: #8f8c8a;
  min-height: 100vh;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  font-size: calc(10px + 2vmin);
  color: white;
}

.link {
  color: #61dafb;
}
