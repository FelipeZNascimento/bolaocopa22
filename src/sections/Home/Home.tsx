import React from 'react';
import { Loading } from '@omegafox/components';
import styles from './Home.module.scss';
import logo from 'img/spinner.png';

export const Home = () => {
  return (
    <div className={styles.app}>
      <Loading image={logo} text="" />
    </div>
  );
};
