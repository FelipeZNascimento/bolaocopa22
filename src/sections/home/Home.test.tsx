import React from 'react';
import { render, screen } from '@testing-library/react';
import { Home } from './Home';

describe('Home', () => {
  it('should render', () => {
    const { container } = render(<Home />);
    expect(container).toMatchSnapshot();
  });

  it('renders learn react link', () => {
    render(<Home />);
    const linkElement = screen.getByText(/learn react/i);
    expect(linkElement).toBeInTheDocument();
  });
});
