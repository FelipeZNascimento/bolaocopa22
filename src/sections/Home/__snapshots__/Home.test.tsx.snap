// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should render 1`] = `
<div>
  <div
    class="app"
  >
    <header
      class="header"
    >
      <p>
        Edit 
        <code>
          src/App.tsx
        </code>
         and save to reload.
      </p>
      <a
        class="link"
        href="https://reactjs.org"
        rel="noopener noreferrer"
        target="_blank"
      >
        Learn React
      </a>
    </header>
  </div>
</div>
`;
